aii-core/src/main/perl/Shellfe.pm

```diff
@@ -229,19 +231,23 @@ sub app_options
          HELP    => 'where to store lock files',
          DEFAULT => "/var/lock/quattor" },
 
+       { NAME    => 'parallel=i',
+         HELP    => 'run commands on hosts in parallel ',
+         DEFAULT => 0 },
+
+       { NAME    => 'logpid',
+         HELP    => "Add process ID to the log messages (disabled by default, unless parallel is > 1)",
+         DEFAULT => undef },
+
          # Options for osinstall plug-ins
        { NAME    => 'osinstalldir=s',
          HELP    => 'Directory where Kickstart files will be installed',
          DEFAULT => '/osinstall/ks' },
 
          # Options for DISCOVERY plug-ins
-         { NAME => 'dhcpconf=s',
-           HELP => 'name of dhcp configuration file',
-           DEFAULT => '/etc/dhcpd.conf' },
-
-         { NAME => 'restartcmd=s',
-           HELP => 'how to restart the dhcp daemon',
-           DEFAULT => 'service dhcpd restart' },
+       { NAME => 'dhcpcfg=s',
```

Comment:
@jouvin these options still exist in the AII::DHCP module, I only removed them from the AII::Shellfe. I don't find where they should have been passed to aii-dhcp nor AII::DHCP. Imho this could only be done by setting this is /etc/aii/aii-dhcp.conf. Can you verify this? 
This is still true for the new code by default. By default AII::DHCP will get his config from   /etc/aii/aii-dhcp.conf unless specified otherwise with the new `dhcpcfg` option.